tron/core/job.py

```diff
@@ -96,6 +97,7 @@ def __init__(
         time_zone=None,
         service=None,
         deploy_group=None,
+        expected_runtime=None
```

Comment:
Why *not* give this a sane default like 24h ?

It doesn't auto-kill, but any job/action can get stuck for lots of reasons.

Reply:
Yeah, make sense.